### PR TITLE
Add beam info to header

### PR DESCRIPTION
**Pull request by e-koch**
From #17.

### CODE REVIEW

**Comment by keflavich** [comment]
FWIW, I've been doing
`header.update(beam.to_header_keywords)`
in code that needs this.  Maybe we should use that instead of the internal 3 lines here?

Also, perhaps make the 'copy' an option - add a `copy=True` kwarg.  Inplace updates are sometimes needed.

**Comment by e-koch** [comment]
@keflavich -  Oh, I didn't know ```beam.to_header_keywords``` was there! I've made some updates utilizing it.

File: radio_beam/beam.py [hunk 1]
```diff
@@ -149,6 +149,33 @@ def from_aips_header(cls, hdr):
...
+        '''
+
+        if copy:
+            copy_header = header.copy()
+
+            copy_header.update(self.to_header_keywords())
+
+            return copy_header
+
+        header.update(self.to_header_keywords())
```
Comment (keflavich) [inline radio_beam/beam.py]: any reason not to just do this?
```
     if copy:
         header = header.copy()

       header.update(self.to_header_keywords())

       return header
```

Comment (e-koch) [inline radio_beam/beam.py]: Looks good, but does the header need to be returned if it isn't being copied?

Comment (keflavich) [inline radio_beam/beam.py]: for consistency's sake, I think it's nice to have the function always return something

**Comment by e-koch** [comment]
@keflavich - I've simplified it a bit. Ready for merge?

**Pull request merged**